index.html

```diff
@@ -215,7 +215,50 @@ <h2 class="holder-title">
     <h3 class="last-title-container">
         I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
     </h3>
-    </div>
+    <form class="form-block">
+        <section>
```

Comment:
- [ ] `section` does not have any semantic meaning here. kindly remove that.
`You can check` [here](https://www.w3schools.com/tags/tag_section.asp) `when to use the section tag`